### DISCUSSION TITLE
I cannot launch minecraft from my ide to test my mods. 1.16.5

### DISCUSSION THREAD

**Discussion by SekySmirfBoi** [FabricMC/fabric-api#1773] [Player support] [answered]
These are my errors
![errors](https://user-images.githubusercontent.com/83304246/137176587-71d6d51b-4789-4df9-9970-fa948bcf5c4f.png)

**Comment by Juuxel** [comment, marked as answer]
The error is cut off in the screenshot, but it seems that the issue is with your mod ID. Remember that those can't contain capital letters, for example. The exact error message (if you scroll to the right) should tell you what exactly is wrong with the ID.

**Reply by SekySmirfBoi** [reply]
Yes this helped thank you
